packages/relayer/processor/config.go

```diff
@@ -140,7 +140,7 @@ func NewConfigFromCliContext(c *cli.Context) (*Config, error) {
 	return &Config{
 		hopConfigs:                         hopConfigs,
 		ProcessorPrivateKey:                processorPrivateKey,
-		SrcSignalServiceAddress:            common.HexToAddress(c.String(flags.SrcSignalServiceAddress.Name)),
+		SrcSignalServiceAddress:            common.HexToAddress(c.String(flags.DestSignalServiceAddress.Name)),
```

Comment:
should we rename this to DestSignalServiceAddress?

Reply:
Will pick up destination event, `SrcSignalServiceAddress` suggests changing it as well, otherwise it will easily cause misunderstanding.

Reply:
@mask-pp i dont understand your comment, i think Roger is correct?